### ISSUE TITLE
run as admin required?

### ISSUE THREAD

**Issue by UAlbanyErich** [ExLibrisGroup/SpineOMatic#44]
I don't normally use SpineOMatic, but in troubleshooting [a connection issue](https://github.com/ExLibrisGroup/SpineOMatic/issues/43), I "installed" it (to `c:\Spine`) using a local admin account.  `SpineLabeler.exe` wouldn't open for my normal, unprivileged account, but it did open if my account was given admin rights.  I adjusted the NTFS security so all users have "full control" to the entire folder, but `SpineLabeler.exe` still won't open if my account is unprivileged.  Windows shows a very brief "wait circle", and if I'm watching processes, I can see the `SpineLabeler.exe` appear and then immediately disappear.  

Any ideas why?  Are there permissions elsewhere (e.g. registry) that need to be granted?

Thanks.

**Comment by mgobat** [comment]
@UAlbanyErich, it should not require admin privileges. Does the same problem happen if installed in a user directory rather than in the C root?

**Comment by UAlbanyErich** [comment]
Yes, it does.  Copying the folder or downloading a new copy of the .exe from here into a folder within the unprivileged user profile, and it attempting to start it as that unprivileged user does the same 1 second wait circle and nothing.  Right-clicking -> run as admin and entering admin credentials and it starts right up.

**Comment by SSWinger** [comment]
I get  a message saying windows protected your PC and I have to click More info and then run anyway. Just had Windows 11 put on my computer not sure if that is part of it. Also having issues with prefixes, but that was prior to the Windows 11 update.
